lib/registries/addon/drafts/draft/route.ts

```diff
@@ -31,6 +31,8 @@ export default class DraftRegistrationRoute extends Route {
                 draftId,
                 { adapterOptions: { include: 'branched_from' } },
             );
+            const draftRegistrationSubjects = yield draftRegistration.loadAll('subjects');
+            draftRegistration.set('subjects', draftRegistrationSubjects);
```

Comment:
Have to fetch all the current subjects here when we initialize the draft manager. Otherwise, validation for subjects wouldn't work when you go directly to the review page, because when the validation function is run the subjects are not loaded yet.